src/html/fetch-content.js

```diff
@@ -55,7 +55,7 @@ async function fetchContent(context, {
   }
 
   try {
-    logger.debug(`loading from content proxy: ${contentProxyUrl}`);
+    logger.info(`loading from content proxy: ${contentProxyUrl}`);
```

Comment:
Note, for easier debugging, it would be great to know the content proxy URL - is that ok with everybody, it hopefully does not reveal anything it shouldn't.